### ISSUE TITLE
The preview section is not rendering. (0.3.9)

### ISSUE THREAD

**Issue by Jaswetz** [SC5/sc5-styleguide#562]
I updated today and everything builds out fine but all the preview sections are empty. This happens in two different projects on update. The last version 0.3.8, worked fine.

![2_2_body_text_-_my_styleguide](https://cloud.githubusercontent.com/assets/1703954/7058062/bc8ef5e8-de11-11e4-8094-0d975dd51467.png)

```
[17:02:09] Using gulpfile ~/Sites/flat-projects/gulp-starter/gulpfile.js
[17:02:09] Starting 'browser-sync'...
[17:02:09] Finished 'browser-sync' after 22 ms
[17:02:09] Starting 'styleguide:generate'...
[17:02:09] Starting 'styleguide:applystyles'...
[BS] Access URLs:
 -------------------------------------
       Local: http://localhost:3000
    External: http://192.168.1.13:3000
 -------------------------------------
          UI: http://localhost:3001
 UI External: http://192.168.1.13:3001
 -------------------------------------
[BS] Serving files from: build/development
[17:02:09] Finished 'styleguide:applystyles' after 388 ms
Express server listening on port 5000
[17:02:09] Finished 'styleguide:generate' after 844 ms
[17:02:09] Starting 'styleguide'...
[17:02:09] Finished 'styleguide' after 11 μs
[17:02:09] Starting 'watch'...
[17:02:09] Finished 'watch' after 57 ms
Socket connection established (id: 6sQMH9mPiYoGQEkQAAAA)
GET /sectio 200 10.726 ms - 1688
GET /styleguide_at_rules.css 200 4.285 ms - -
GET /js/app.js 200 7.050 ms - 31743
GET /css/app.css 200 6.854 ms - 188864
GET /js/vendor.js 200 45.836 ms - 1481434
GET /views/main.html 200 2.208 ms - 2833
GET /views/404.html 200 1.414 ms - 32
GET /styleguide.json 200 3.173 ms - 30667
GET /views/partials/design.html 200 2.314 ms - 2280
GET /assets/img/sc5logo.svg 200 1.801 ms - 1935
GET /assets/fonts/fontawesome-webfont.woff?v=4.2.0 200 2.242 ms - 65452
Socket connection established (id: hYpfS6eEXb-3NNSfAAAB)
GET /views/partials/variable.html 200 1.507 ms - 589
GET / 200 1.123 ms - 1688
GET /js/vendor.js 304 3.276 ms - -
GET /js/app.js 304 3.232 ms - -
GET /styleguide_at_rules.css 304 1.771 ms - -
GET /css/app.css 304 1.508 ms - -
GET /views/main.html 304 0.704 ms - -
GET /overview.html 200 0.918 ms - 170
GET /assets/img/sc5logo.svg 304 0.514 ms - -
GET /styleguide.json 304 1.559 ms - -
GET /views/partials/design.html 304 0.732 ms - -
GET /assets/fonts/fontawesome-webfont.woff?v=4.2.0 304 0.908 ms - -
Socket connection established (id: gsyaEi35MPp67eECAAAC)
GET /views/partials/variable.html 304 0.675 ms - -
GET /views/sections.html 200 2.029 ms - 114
GET /views/partials/section.html 200 0.954 ms - 2878
GET /styleguide.css 200 3.268 ms - 15366
GET /css/styleguide_helper_elements.css 200 3.125 ms - 2091
GET /styleguide_pseudo_styles.css 200 3.844 ms - 257
```

**Comment by hannu** [comment]
It seems that new variables parser detects false positives for variable declarations. We are currently working on this issue.

**Comment by hannu** [comment]
Due to vast amount of issues with the newer gonzales, sc5-styleguide 0.3.10 is now released containing older gonzales (3.0.0-12).

Please re-open this issue if this issue still exists with the 0.3.10
